.gitignore

```diff
@@ -1,2 +1,3 @@
 bin
 python/version.py
+__pycache__
```

Comment:
@jjsjann123 You cool with this?

Reply:
good catch! I'm gonna need that once we have separate setup.py in nvfuser.
But I'm actually curious why you would need it here?